tests/functional/api/test_groups.py

```diff
@@ -78,40 +102,45 @@ def test_it_returns_http_404_with_invalid_forwarded_user_format(self, app, auth_
 @pytest.mark.functional
 class TestAddMember(object):
 
-    def test_it_returns_http_204_when_successful(self, app, user, group, auth_client_header):
-        res = app.post_json("/api/groups/{pubid}/members/{userid}".format(pubid=group.pubid, userid=user.userid),
-                            headers=auth_client_header)
+    def test_it_returns_http_204_when_successful(self, app, third_party_user, third_party_group, auth_client_header):
```

Comment:
MOST of these test changes aren't related to meaningful PR changes, but because I had to adjust a bunch of fixtures and de-couple some authority things.